com.unity.render-pipelines.core/ShaderLibrary/Common.hlsl

```diff
@@ -1,6 +1,8 @@
 #ifndef UNITY_COMMON_INCLUDED
 #define UNITY_COMMON_INCLUDED
 
+#pragma warning (disable : 3205) // conversion of larger type to smaller
```

Comment:
Rather than disabling for the world, I will suggest to 
#pragma warning (disable : 3205)  / #pragma warning (enable : 3205) on each required file